src/material-examples/tab-group-async/tab-group-async-example.html

```diff
@@ -3,7 +3,18 @@
 </ng-container>
 
 <mat-tab-group>
-  <mat-tab *ngFor="let tab of asyncTabs | async">
+    <!--
+
+        TODO for review: In that case it's quite weird because with the async pipe
```

Comment:
We should talk about this one in the framework meeting

Reply:
PR that should solve this is here: https://github.com/angular/components/pull/16373. Needs review from Igor.

Reply:
@devversion that link seems to just link this issue rather than one in `angular/angular`?

Reply:
The right PR link is https://github.com/angular/angular/pull/31371. That PR has been merged and released in `9.0.0-next.10`.